### DISCUSSION TITLE
2take1 is detecting tokens as spoofed when token < 10^16

### DISCUSSION THREAD

**Discussion by lonelybud** [YimMenu/YimMenu#3314] [General]
i can confirm when using custom token and joining a 2take1 user. Also once by chance i got my token 1404178021824864 (not spoofed) by the game but 2take1 user marked me as spoofed. I was so pissed.

**Comment by snowleopardd** [comment]
u should post this on the issues tab ngl

**Reply by lonelybud** [reply]
i cant confirm but the guy told me it was 2take1. It is terrifying to know what logic they are using. Let it be discussion

**Reply by maybegreat48** [reply]
This isn't an issue
